src/libs/antares/study/include/antares/study/parts/hydro/prepro.h

```diff
@@ -95,8 +95,9 @@ class PreproHydro
     ** \param folder The source folder (ex: `input/hydro/prepro`)
     ** \return A non-zero value if the operation succeeded, 0 otherwise
     */
-    bool loadFromFolder(Study& s, const AreaName& areaID, const char folder[]);
+    bool loadFromFolder(Study& s, const AreaName& areaID, const std::string& folder);
```

Comment:
Is it possible (or usefull) to use std::filesystem::path for folder ?

Reply:
will be done in another PR